pylint/plugins/hass_enforce_greek_micro_char.py

```diff
@@ -0,0 +1,48 @@
+"""Plugin for checking correct micro unicode card is used."""
+
+from __future__ import annotations
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.lint import PyLinter
+
+
+class HassEnforceGreekMicroCharChecker(BaseChecker):
+    """Checker for micro char."""
+
+    name = "hass-enforce-greek-micro-char"
+    priority = -1
+    msgs = {
+        "W7452": (
+            "Constants with the mico sign must be encoded as U+03BC (\u03bc), not as U+00B5 (\u00b5)",
```

Comment:
I think the message here is a bit off. The name of the character is mu, not micro. Micro is the unit prefix.
Maybe something like this:
```suggestion
            "Constants with a micro unit prefix must be encode mu as U+03BC (\u03bc), not as U+00B5 (\u00b5)",
```